pkg/controller/controller_instance.go

```diff
@@ -339,6 +455,18 @@ func (c *controller) reconcileServiceInstanceAdd(instance *v1beta1.ServiceInstan
 		return nil
 	}
 
```

Comment:
My proposed additional change: add `backoffRequired map[string]struct{}{}` to `retryQueue` struct, and check it at this line. If the flag is set, invoke `setNextOperationRetryTime` there instead of all other places.
All current usages of `setNextOperationRetryTime` replace with `setBackoffRequired(true)`.
Same change for `reconcileServiceInstanceUpdate` method.